repo/fsrepo/fsrepo.go

```diff
@@ -562,7 +562,9 @@ func (r *FSRepo) GetStorageUsage() (uint64, error) {
 
 	var du uint64
 	err = filepath.Walk(pth, func(p string, f os.FileInfo, err error) error {
-		du += uint64(f.Size())
+		if f != nil {
```

Comment:
The stat is called from the `filepath.Walk` (https://golang.org/src/path/filepath/path.go#L368). The specific `walkFn` call here doesn't return any err value unless if a `nil` stat is considered err.